app/assets/stylesheets/tylium/modules/_dots-menu.scss

```diff
@@ -1,55 +1,48 @@
-.dots-container {
-  background-color: #e9ecef; // match BS4 breadcrumb bg color
-  display: flex;
-  justify-content: space-between;
+.dots-dropdown {
   align-items: center;
-  padding-right: 1rem;
+  border: 1px solid darken($borderColor, 2%);
+  border-radius: 50px;
+  cursor: pointer;
+  display: inline-flex;
+  height: 1.75rem;
+  justify-content: center;
+  margin-left: auto;
+  transition: all 0.2s ease-in-out;
+  width: 1.75rem;
+
+  .dots-dropdown-header {
+    font-weight: 400 !important;
+    
+    &:hover {
+      color: $linkColor !important;
+      background-color: initial !important;
+    }
 
-  .breadcrumb {
-    width: calc(100% - 35px);
+    &:not(:first-child) {
```

Comment:
What's this change for?

Reply:
This gives header items in the dropdown menu some extra styling if the header is not at the top of the menu (we don't want double borders)